packages/mysql-community-8.0-mroonga/apt/debian-bookworm/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG FROM=debian:bookworm
+FROM ${FROM}
+
+RUN \
+  echo "debconf debconf/frontend select Noninteractive" | \
+    debconf-set-selections
+
+ENV \
+  DEBIAN_FRONTEND=noninteractive \
+  MYSQL_SERVER_VERSION=mysql-8.0
+
+ARG DEBUG
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "-qq") && \
+  apt update ${quiet} && \
+  apt install -y -V --no-install-recommends ${quiet} \
+    ca-certificates \
+    lsb-release \
+    wget && \
+  wget https://packages.groonga.org/$(lsb_release --id --short | tr 'A-Z' 'a-z')/groonga-apt-source-latest-$(lsb_release --codename --short).deb && \
```

Comment:
We need Apache Arrow repository too.

Reply:
Thank you for your comment.
I've fixed it.